monitor/data_monitoring.py

```diff
@@ -141,6 +136,9 @@ def generate_report(self, days_back: Optional[int] = None, test_runs_amount: Opt
             output_data['test_runs_totals'] = test_runs_totals
             output_data['coverages'] = models_coverages
             output_data['lineage'] = lineage.dict()
+            output_data['report_generator_user_id'] = tracking.anonymous_user_id
```

Comment:
It just aligns a bit better to me because the function is `generate_report`, what do you think?
Not that critical though.

Reply:
Not super critical, I do think that we maybe should have tracking dict passed to the report instead of passing it globally - 
report[tracking] = {'posthog_api_key': ..., 'anonymous_warehouse_id': ..., 'anonymous_report_generator_id'}

If you do the change, please add 'anonymous' as a prefix to the variable names. Also, please note that not all the dbt databases are called warehouses so maybe it should be an adapter?

Reply:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/30181361/183399111-3dfe7b1c-9cb6-4fac-adcc-1a1139d01857.png">
I think it's clearer than `adapter` which is the piece of technology that abstracts the differences between the warehouses themselves.